mrt/static/js/printouts.js

```diff
@@ -21,7 +21,7 @@ $(function () {
         var job_row = $('#' + this.id);
         job_row.addClass('success');
         job_row.find('.status').text(STATUS.FINISHED);
-        var html = $('<a>').attr('href', result).text('Download file');
+        var html = $('<b>').text('Success - please refresh');
```

Comment:
We should be able to easily check if the value of `result` is a link or not, and either display a link or the message in `result`. This would be far more convenient than forcing the user to refresh.